src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs

```diff
@@ -234,7 +236,7 @@ internal IEnumerable<BluetoothGattService> GetServices(BluetoothGattClient clien
                 return true;
             };
 
-            int err = Interop.Bluetooth.BtGattClientForeachServices(_handle, cb, IntPtr.Zero);
+            int err = Interop.Bluetooth.BtGattClientForeachServices(_handle, _serviceForeachCallback, IntPtr.Zero);
```

Comment:
`BtGattClientForeachServices` is not async method, so need not change.
anyway it also no matter

Reply:
Thank you for comment. I want to leave it as it is.